.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: ".github/workflows" # Location of package manifests
+    schedule:
+      interval: "monthly"
+    reviewers:
+      - "zacharyburnett"
```

Comment:
@zacharyburnett just to confirm. You wanted me to add this right? I don't want to sneak this in post-approval if not.

Reply:
yes indeed, please!